ovos_PHAL_plugin_commonIOT/vui.py

```diff
@@ -1,7 +1,84 @@
 from ovos_workshop.app import OVOSAbstractApplication
-
+from ovos_workshop.decorators import intent_handler
+from ovos_bus_client import Message
+from ovos_utils.log import LOG
 
 class IOTVoiceInterface(OVOSAbstractApplication):
     def __init__(self, bus=None):
         super().__init__(skill_id="ovos.iot", bus=bus)
         # TODO - pretend this is a skill and implement intents
+
+    @intent_handler("light.get.brightness.intent")
+    def light_get_brightness_intent(self, message):
+        device = message.data.get("device")
+        response = self.bus.emit(Message("ovos.iot.device.get.brightness", data={"device": device}))
```

Comment:
in here you should use some method to capture the emitted bus event, self.bus.emit does not return anything

maybe wait_response?